validator/src/main.rs

```diff
@@ -1883,6 +1897,34 @@ pub fn main() {
                 .help("Allow contacting private ip addresses")
                 .hidden(true),
         )
+        .arg(
+            Arg::with_name("retransmit_sent_max_duplicate_count")
+                .long("retransmit-sent-max-duplicate-count")
+                .takes_value(true)
+                .default_value(&default_retransmit_sent_max_duplicate_count)
+                .hidden(true),
+        )
+        .arg(
+            Arg::with_name("retransmit_shreds_received_lru_cache_size")
+                .long("retransmit-shreds-received-lru-cache-size")
+                .takes_value(true)
+                .default_value(&default_retransmit_shreds_received_lru_cache_size)
+                .hidden(true),
+        )
+        .arg(
+            Arg::with_name("retransmit_packet_batch_threshold")
+                .long("retransmit-packet-batch-threshold")
+                .takes_value(true)
+                .default_value(&default_retransmit_packet_batch_threshold)
+                .hidden(true),
+        )
+        .arg(
```

Comment:
Do we see a use-case for the users running validators with different values for these arguments?

There is usually some risk with adding these arguments at a high level to main validator binary. cc @mvines since previously we had discussions over these flags.

Reply:
I was introducing these to make it easier to test the impact of changing these values without recompiling. I didn't intend for them to be generally used by validator operators. They're marked as hidden. I could prefix these flags with something. I see "experimental-poh-pinned-cpu-core". Thoughts?